src/priority_fee.rs

```diff
@@ -403,7 +572,7 @@ mod tests {
         set_global_default(client)
     }
 
-    #[tokio::test]
+    #[tokio::test(flavor = "current_thread")]
```

Comment:
what is this change getting us?

Reply:
ah, I guess nothing except more explicit definition that single thread profile will be used. I did not see if originally but the documentation states that the default test profile is single-threaded. 

https://docs.rs/tokio-macros/latest/tokio_macros/attr.test.html

WIll remove